src/policy/rbf.h

```diff
@@ -6,6 +6,11 @@
 #define BITCOIN_POLICY_RBF_H
 
 #include <txmempool.h>
+#include <consensus/validation.h>
```

Comment:
Done

Reply:
Actually decided to remove this dependency because I realized it's not the right place to be deciding what `TxValidationResult` to return (should be in validation.cpp)
But yes, sorted includes :salute: